### PR TITLE
With react-table (for dashboard tables also)

### LINKED ISSUE #172: Pagination on the Livepeer.com Dashboard streams page list
https://livepeer.com/app/user

- 100 streams per page
- Add "previous page" and "next page" buttons to the bottom of the list. 
- The buttons should visually look the same as the previous and next page buttons that @darkdarkdragon created in [this PR](https://github.com/livepeer/livepeer-com/pull/99), **but they should be at the bottom of the list centered side by side**

- If there are less than 100 streams for the user, do not show the buttons
- If there is no next page of streams, do not show the "next page"
- If there is no previous page of streams, do not show the "previous page button"

Relates to https://github.com/livepeer/livepeer-com/issues/173

### LINKED ISSUE #173: Sort the Livepeer.com dashboard streams page by name, createdAt, and lastActive
https://livepeer.com/app/user

- Add ability to sort Asc and Desc for the name, created at and last seen at columns
- default to show the most recent last seen streams first
- Use the same arrow styling seem on https://livepeer.com/app/admin/streams 

This needs work with https://github.com/livepeer/livepeer-com/issues/172. Example: If I have more than 100 streams and I sort by name, all pages need to work as expected in alphabetical order.

### PR DESCRIPTION
**Pull request by julianbenegas**
**What does this pull request do? Explain your changes. (required)**

- Add a generic table component that uses `react-table`

**How did you test each of these updates (required)**

I've tested by using the new table component, sorting stuff, selecting stuff, trying to break stuff...

**Does this pull request close any open issues?**

- closes #173 
- closes #172 

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/www/components/Checkbox/index.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Box } from "@theme-ui/components";
```
Comment (adamsoffer) [inline packages/www/components/Checkbox/index.tsx]: Let's combine these imports `import { Box, Flex } from '@theme-ui/components'`.

**Review by adamsoffer — CHANGES_REQUESTED** [review]
@julianbenegas awesome work. I don't have 100 streams created so I'm unable to test the pagination feature but everything looks GREAT.

**Review by wohlner — CHANGES_REQUESTED** [review]
For testing purposes can you change the limit to like 5 streams or something else tiny so that we can test the pagination feature?

**Comment by wohlner** [comment]
I am not sure sorting is working. I am testing on https://livepeer-com-git-jb-with-react-table.livepeerorg.vercel.app/app/user. Look at the created at column, it's not in the right order when I sort it. 
Screen Shot 2021-01-29 at 8 59 09 AM
Same goes for the name column when I sort it.
Screen Shot 2021-01-29 at 9 01 12 AM

**Review by wohlner — CHANGES_REQUESTED** [review]
This looks great, nice job. All the sorting, deleting, creating, and pagination works. I'd like to request two small tweaks.

1. Right now in your PR and in production on livepeer.com, we default with the most recently active stream first. Instead, let's sort with the most recently **created** stream first. It was a weird user experience for me to create a new stream and then have to click to the second page of the streams list to find it (since it was _unseen_ and never active.
2. When the page loads, can you please show how the data is being organized by default, what column is controlling the sorting? Make a down arrow next to the Created At column header like the image below.
Screen Shot 2021-01-29 at 4 07 22 PM

**Review by wohlner — APPROVED** [review]
Looks great. Just return the stream list to the original length, and this is good to go. Nice work!

**Comment by julianbenegas** [comment]
Thank you @wohlner. Page size should be 100 now. Waiting for @adamsoffer before merging!

**Review by adamsoffer — APPROVED** [review]
LGTM!

**Pull request merged**